### DISCUSSION TITLE
Assets module adding default keyword to imports

### DISCUSSION THREAD

**Discussion by Aveline-art** [webpack/webpack#16202] [Q&A]
# Bug report

**What is the current behavior?**
When importing assets, via `import sky from '../assets/sky.png'`, the compiled file adds the .default keyword whenever the asset is used.

**If the current behavior is a bug, please provide the steps to reproduce.**
[See this for a more thorough write up (not by me)](https://stackoverflow.com/questions/66047474/webpack-5-how-to-import-assets-as-es6-module)

**What is the expected behavior?**
I would like for imports to work without using `* as sky from '..asset/sky.png'` to work around this, since the documentation does not indicate that this is needed. Not sure if this is a feature or a bug when outputting ES6, honestly, but it was difficult for me to diagnose what I was doing wrong with the import.

**Other relevant information:**
webpack version:  5.47.0
Node.js version:  16.16.0
Operating System: MacOS (m1)
Additional tools:

**Comment by alexander-akait** [comment]
@Aveline-art Sounds like you use typescipt and doesn't setup it, please provide configuration and I will show you how to fix it

**Comment by Aveline-art** [comment]
Hmm, that's bizarre then, since I use the same ts config as with a different project but did not encounter this issue. Nevertheless, here you go!

`tsconfig.json`
```json
{
  "compilerOptions": {
    "target": "ES6",
    "module": "CommonJS",
    "rootDir": "./src",
    "outDir": "./dist",
    "jsx": "react-jsx",
    "moduleResolution": "node",
    "jsxImportSource": "preact",
    "types": ["preact"]
  }
}
```

**Reply by alexander-akait** [reply]
hmm, maybe you have babel? Do you have something special in configuration(s)? Double check this, because if we have this problem, it has already been in the top of the problems.

**Reply by Aveline-art** [reply]
You're absolutely right. I do use babel with this! In my other project, I recently cut out babel as the middleman, which could explain the difference. I'll get back to you once I take a look at my babel.

**Reply by Aveline-art** [reply]
Got it. It was this tsconfig setting that I was missing: https://www.typescriptlang.org/tsconfig#esModuleInterop
